libs/shared/src/packages/sidesheet/src/lib/sidesheet/tabs/modelviewer/ModelViewerTab.tsx

```diff
@@ -0,0 +1,89 @@
+import { Widget } from '@equinor/fusion-framework-react-app/widget';
+import { TagOverlay } from '@cc-components/modelviewer';
+import { ModelViewerConfig } from '@cc-components/modelviewer';
+import { Icon, Progress } from '@equinor/eds-core-react';
+import { tokens } from '@equinor/eds-tokens';
+
+import styled from 'styled-components';
+
+export const NoResourceData = styled.div`
+  text-align: center;
+  padding: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 10px;
+`;
+
+export const InfoText = styled.h3`
+  margin: 0;
+`;
+
+export type ModelViewerTabProps = {
+  TagOverlay: string[] | TagOverlay[] | undefined;

```

Comment:
Why is this PascalCase and not camelCase?

Reply:
Refactored